central/main.go

```diff
@@ -274,6 +275,11 @@ func main() {
 
 	premain.StartMain()
 
+	if err := continuousprofiling.SetupContinuousProfilingClient(continuousprofiling.DefaultConfig(),
```

Comment:
```suggestion
	if err := continuousprofiling.SetupClient(continuousprofiling.DefaultConfig(),
```
Wouldn't that read a bit better?

Reply:
Yeah that reads much better. I've changed in the last commit.